sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Queues/tests/QueueTriggerTests.cs

```diff
@@ -96,6 +96,33 @@ private async Task TestBindToString(string expectedContent)
             Assert.AreEqual(expectedContent, result);
         }
 
+        [Test]
+        public async Task QueueTrigger_IfBoundToParameterBindingData_Binds()
+        {
+            string expectedContent = Guid.NewGuid().ToString();
+            await TestBindToParameterBindingData(expectedContent);
+        }
+
+        [Test]
+        public async Task QueueTrigger_IfBoundToParameterBindingDataAndMessageIsEmpty_Binds()
+        {
+            await TestBindToParameterBindingData(string.Empty);
+        }
+
+        private async Task TestBindToParameterBindingData(string expectedContent)
+        {
+            // Arrange
+            var queue = await CreateQueue(queueServiceClient, QueueName);
+            await queue.SendMessageAsync(expectedContent);
+
+            // Act
+            string result = await RunTriggerAsync<ParameterBindingData>(typeof(BindToParameterBindingData),
```

Comment:
Need to change this type and fixup the test?
```suggestion
            ParameterBindingData result = await RunTriggerAsync<ParameterBindingData>(typeof(BindToParameterBindingData),
```

Reply:
Yup ☠️